projects/components/src/utils/rest/rest-query-search.client.ts

```diff
@@ -89,6 +90,10 @@ export class RestQueryService extends VcdApiClient {
             multisite = options.multisite;
         }
 
+        if (options?.format) {
+            toSend.format(options?.format);
```

Comment:
No need for  the `?` in `options?.format` here since you just checked it on the line above.